pages/common/jhipster.md

```diff
@@ -0,0 +1,36 @@
+# jhipster
+
+> Generates a monolithic or microservices project after prompting user.
+> More information: <https://www.jhipster.tech/>.
+
+- Generate a simple full-stack project (monolithic or microservices):
+
+`jhipster`
+
+- Generate a simple frontend project with `--skip-server` or a simple backend project with `--skip-client`:
```

Comment:
This is still kinda problematic, as it's a mesh of two examples

Reply:
I've splitted them again but removed the one about `docker-compose`